SCRUM-6

mtakbaba opened the pull request.

mtakbaba committed:
Create asd

mtakbaba committed:
Update index.html

mtakbaba committed:
Update vite.config.js

mtakbaba committed:
Delete asd

mtakbaba committed:
Update main.py

mtlgkbb20 committed:
Add files via upload

mtlgkbb20 committed:
Update KpiDefinition.java

mtlgkbb20 committed:
Delete spanish_tutor_backend/api/user_notes.py

mtlgkbb20 committed:
Delete spanish_tutor_backend/api/user_interests.py

mtlgkbb20 committed:
Delete spanish_tutor_backend/api/user.py

mtlgkbb20 committed:
Delete spanish_tutor_backend/models.py

mtakbaba committed:
Merge pull request #22 from mtlgkbb20/!!

!!

mtakbaba committed:
asd

sd

mtakbaba committed:
Update CSharpParserBase.java

mtakbaba committed:
Update CSharpParserBase.java

mtakbaba committed:
SCRUM-10 for issue

mtakbaba committed:
SCRUM-11 another issue

mtakbaba committed:
SCRUM-12 testing_right_now

mtakbaba commented on CSharpLexerBase.java:
```diff
@@ -0,0 +1,104 @@
+    }
+
+    protected void OnInterpolatedVerbatiumStringStart()
+    {
```
lets  change here please

mtakbaba commented on CSharpLexerBase.java:
```diff
@@ -0,0 +1,104 @@
+
+    protected void OnColon()
+    {
+
```
not a good name

mtakbaba commented on CSharpLexerBase.java:
```diff
@@ -0,0 +1,104 @@
+        curlyLevels.pop();
+    }
+
+    protected boolean IsRegularCharInside()
```
here is nice

mtakbaba replied:
commented for review

mtakbaba reviewed (commented):
plz have a look

mtakbaba committed:
SCRUM-6

mtakbaba commented on CSharpLexerBase.java:
```diff
@@ -0,0 +1,104 @@
+        {
+            curlyLevels.push(curlyLevels.pop() + 1);
+        }
+    }
```
have a look

mtlgkbb20 commented on CSharpLexerBase.java:
```diff
@@ -0,0 +1,104 @@
+    print ("test");
+
+    protected void OnInterpolatedRegularStringStart()
+    {
```
I am checking this file

mtlgkbb20 commented on CSharpLexerBase.java:
```diff
@@ -0,0 +1,104 @@
+        verbatium = (interpolatedVerbatiums.size() > 0 ? interpolatedVerbatiums.peek() : false);
+    }
+
+    protected void OnCloseBraceInside()
```
fix here

mtlgkbb20 reviewed (commented):
hereee

mtlgkbb20 commented:
.

mtlgkbb20 commented:
löaşlds

mtlgkbb20 commented on CSharpLexerBase.java:
```diff
@@ -0,0 +1,104 @@
+                }
+                ind++;
+            }
+            if (switchToFormatString)
```
please take this out

mtlgkbb20 commented on CSharpLexerBase.java:
```diff
@@ -0,0 +1,104 @@
+            {
+                mode(CSharpLexer.INTERPOLATION_FORMAT);
+            }
+        }
```
check

mtlgkbb20 reviewed (commented):
a

mtakbaba committed:
Update CSharpLexerBase.java

fixed problem SCRUM-6

mtlgkbb20 committed:
Modify testfile.txt by removing and adding lines

Removed multiple lines and added a series of new lines with various characters.

mtlgkbb20 committed:
Add multiple lines of text to testfile.txt

mtlgkbb20 committed:
Refactor testfile.txt by removing and adding lines

mtakbaba committed:
Update CSharpLexerBase.java

mtakbaba committed:
Update CSharpLexerBase.java
